crypto/s390xcpuid.S

```diff
@@ -174,7 +225,16 @@ OPENSSL_instrument_bus2:
 	br	%r14
 .size	OPENSSL_instrument_bus2,.-OPENSSL_instrument_bus2
 
+.globl	OPENSSL_vx_probe
+.type	OPENSSL_vx_probe,@function
+.align	16
+OPENSSL_vx_probe:
+	.long	0xe7000000	# vzero %v0
+	.word	0x0044
```

Comment:
It's 6-byte instruction right? Would `.word 0xe700,0x0000,0x004` do? If so, may I suggest this triple-word instead?

Reply:
okay